src/main/java/org/fundacionjala/sfdc/core/driver/DockerChromeBrowser.java

```diff
@@ -13,18 +13,18 @@
  * DockerChromeBrowser class.
  */
 public class DockerChromeBrowser implements Browser {
+    private static final Logger log = Logger.getLogger(DockerChromeBrowser.class.getName());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename the "log" logger to comply with the format "LOG(?:GER)?". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1312)